deb/build/build.sh

```diff
@@ -13,9 +13,15 @@ mkdir -p $D
 # so that multiple builds can go on concurrently
 cp -R $dir/* $D
 
+# Create a description temp file
+sed -i.bak -e 's/^\s*$/./' -e 's/^/ /' $DESCRIPTION_FILE
+
 # Expand variables in the definition
 "$BASE/bin/branding.py" $D/debian
 
+# Rewrite the file 
+mv "$DESCRIPTION_FILE.bak" "$DESCRIPTION_FILE"
```

Comment:
Since the file is changed in place and we don't know the order of the processing the different packaging methods, we have to change the file back after processing the branding.